README.md

```diff
@@ -44,10 +44,10 @@ Move is a programming language for writing safe smart contracts originally devel
 
 ## Move-Powered Blockchains
 
+- [Aptos](https://github.com/aptos-labs/aptos-core) - [Aptos](https://aptos.dev/) strives to be the safest and most scalable layer one blockchain solution (in [mainnet](https://explorer.aptoslabs.com/?network=mainnet)) and offers many [Move examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples).
```

Comment:
Thanks for the contribution, and sorry for the delay! However, the blockchain introduction should be short: examples should be elsewhere.

Reply:
No worries! I moved the examples to the Miscellaneous section.